hologram_dashboard/server/request_handler/hologram_data_availability_reader.cc

```diff
@@ -0,0 +1,63 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#include "hologram_data_availability_reader.h"
+
+namespace wireless_android_play_analytics {
+
+  HologramDataAvailabilityReader::HologramDataAvailabilityReader
+      (absl::string_view path)
```

Comment:
nit: path is too vague. make it more clear about path for what.

You could create a FLAG for the path, and here just parse the FLAG to get the path. There are two advantages:
1) If in the future the path changes, users could update the flag value instead of updating the server binary.
2) In the unit test (of reader and request_handler), you could overwrite the FLAG value to a path only used in tests.